python_modules/libraries/dagster-powerbi/dagster_powerbi/resource.py

```diff
@@ -297,12 +297,15 @@ def build_definitions(
         for content, spec in zip(all_executable_data, all_executable_asset_specs):
             dataset_id = content.properties["id"]
 
+            resource_key = f"power_bi_{self._workspace.workspace_id.replace('-','_')}"
```

Comment:
to potentially clarify what this id is, since this identifier will show up in the logs when resources are initialized

```suggestion
            resource_key = f"power_bi_workspace_{self._workspace.workspace_id.replace('-','_')}"
```